foxy/distribution.yaml

```diff
@@ -4326,6 +4326,14 @@ repositories:
       url: https://github.com/ros-controls/ros2_controllers.git
       version: foxy
     status: developed
+  ros2_example:
```

Comment:
the name of the repo and package are to generic, please check the [package naming guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#guidelines-for-package-naming)

Reply:
The repo name doesn't matter actually, but the package name must follow those guidelines.